.github/workflows/build.yaml

```
@@ -143,3 +146,47 @@ jobs:
           APICULTURIST_GITHUB: ${{ secrets.APICULTURIST_GITHUB }}
           APICULTURIST_TOKEN: ${{ secrets.APICULTURIST_TOKEN }}
           monorepo: ${{ inputs.monorepo }}
+
+  dev_build:
+    runs-on: ubuntu-latest
+    needs: read_package
+    name: ${{ matrix.repos }}
+    strategy:
+      fail-fast: false
+      matrix:
+        repos: ${{fromJSON(needs.read_package.outputs.repoNames)}}
+    steps:
+      - name: Setup workspace
+        uses: qlik-oss/sn-gh-workflows/actions/setup@v3.0.1
+        with:
+          GH_ACCESS_TOKEN: ${{ secrets.GH_ACCESS_TOKEN }}
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN || secrets.NPM_CI_TOKEN }}
+          NPMRC_ENCODED: ${{ secrets.QLIK_NPM_DEV_ENCODED }}
+      - name: Dev build
+        run: pnpm -r --filter ${{ matrix.repos }} build:dev
+      - name: Upload dev build
+        uses: qlik-trial/qmfe-workflows/actions/qmfe-to-aws@v1
```

Skip upload if no system.js files are defined
```suggestion
        uses: qlik-trial/qmfe-workflows/actions/qmfe-to-aws@v1
        if:  ${{ matrix.systemjsFiles != '' }}
```